### PR TITLE
Tidy dead code

### PR DESCRIPTION
**Pull request by skattyadz**

### CODE REVIEW

**Comment by matyikriszta** [comment]
@skattyadz can you comment on what dead code is this removing? has this been replaced by other code? Can you also fix the conflicts?

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -18,10 +18,6 @@ def belongs_to_group?(group)
     current_user.groups.include?(group)
   end
 
-  def can_access?(resource)
```
Comment (skattyadz) [inline app/helpers/application_helper.rb]: This function isn't called by anything

File: app/models/chapter.rb [hunk 2]
```diff
@@ -18,7 +18,6 @@ class Chapter < ActiveRecord::Base
   def self.available_to_user(user)
     return Chapter.all if user.has_role?(:organiser) || user.has_role?(:admin) || user.has_role?(:organiser, Chapter)
     return Chapter.find_roles(:organiser, user).map(&:resource)
-    []
```
Comment (skattyadz) [inline app/models/chapter.rb]: This can't be reached as the line before returns

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (matyikriszta) [reply app/helpers/application_helper.rb]: ok, cool

File: app/models/chapter.rb (hunk 2 above)
Comment (matyikriszta) [reply app/models/chapter.rb]: yep, makes sense

**Comment by matyikriszta** [comment]
@skattyadz coveralls is not liking this one

**Pull request merged**